src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.html

```diff
@@ -49,7 +49,7 @@
           >
             {{ 'profile.dashboard.habits-view-more' | translate }}
           </button>
-          <div class="no-data" *ngIf="this.habitAssignService.habitsInProgress.length === 0">
+          <div class="no-data" *ngIf="!habitAssignService.habitsInProgress.length && !habitsAcquired">
```

Comment:
you can write just `!(habitAssignService.habitsInProgress.length && habitsAcquired)`

Reply:
Thanks, I will change it

Reply:
With !(habitAssignService.habitsInProgress.length && habitsAcquired) It is working in the wrong way 
![image](https://user-images.githubusercontent.com/104850911/220904496-2168d14b-e230-407e-abc0-d8c52062b26c.png)